### PR TITLE
Adding Julia language

### LINKED ISSUE #83: [lang] Add Julia
### Is your feature request related to a problem? Please describe.

This project doesn't have `Julia` language, not even in the pull request I find something that relates to fs!

### Describe the solution you'd like

I love like to add `Julia` language.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Shivansh-yadav13**
### **Description**

Added the Julia language to the list.

This PR fixes #83 

- [x] Yes, I am a Member of **DEVSTRONS'**

**Discord ID** - **Shivansh Yadav#6952**

### CODE REVIEW

File: lang/Julia.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+
+lang: Julia
+
+syntax: julia
+
+title: Julia is a high-level, high-performance, dynamic programming language.
+
+author: Shivansh Yadav
+
```
Comment (Panquesito7) [inline lang/Julia.md:10]:
```suggestion
lang: Julia
syntax: julia
title: Julia is a high-level, high-performance, dynamic programming language.
author: Shivansh Yadav
```

File: lang/Julia.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+---
+
+```julia
+println("Hello World!")
+```
+
+---
+
+### What is Julia
```
Comment (Panquesito7) [inline lang/Julia.md:19]:
```suggestion
## 🤔 What is Julia
```

File: lang/Julia.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+Julia is a new, general purpose, high performance programming language with the tagline **"Looks like Python, feels like Lisp, runs like Fortran”** It is easy to use, powerful and extremally performant.
+
+---
+
+### Why to use Julia
```
Comment (Panquesito7) [inline lang/Julia.md:25]:
```suggestion
## 📈 Why to use Julia
```

File: lang/Julia.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+**In Short**
+
+- Open source
+- Interoperability
+- Reproducible
+- Build to be fast
+- Well suited for data intensive problems
+- Familiar Syntax
+- Dynamic and composable
```
Comment (Panquesito7) [inline lang/Julia.md:41]:
```suggestion
- When using Python and other languages in production and environments it becomes a sort of a burden and a limitation. While Julia is computationally efficient and easy to use for developers.

- Syntactically Julia is quite similar to python so if you write code that does very similar stuff to what it does in python, it does faster in Julia.

- You can also embed all Python, C/C++, and Java code directly inside Julia, run that code and interact with output using Julia.

**In Short**

- Open-source
- Interoperability
- Reproducible
- Build to be fast
- Well suited for data-intensive problems
- Familiar Syntax
- Dynamic and composable
```

File: lang/Julia.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+---
+
+### Setting up Julia development environment
```
Comment (Panquesito7) [inline lang/Julia.md:45]:
```suggestion
## 🛠️ Setting up Julia development environment
```

File: lang/Julia.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+
+#### On Windows & Mac
```
Comment (Panquesito7) [inline lang/Julia.md:47]:
```suggestion
### 💻 On Windows & Mac
```

File: lang/Julia.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+##### Step 1:
```
Comment (Panquesito7) [inline lang/Julia.md:49]:
```suggestion
#### Step 1
```

File: lang/Julia.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+
+> Install Julia 
+
+- Download Julia from [here.](https://julialang.org/downloads/)
```
Comment (Panquesito7) [inline lang/Julia.md:53]:
```suggestion
- Download Julia from the official [Julia website](https://julialang.org/downloads/).
```

File: lang/Julia.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+
+#### On Windows & Mac
+
+##### Step 1:
+
+> Install Julia 
+
+- Download Julia from [here.](https://julialang.org/downloads/) 
+
+##### Step 2:
```
Comment (Panquesito7) [inline lang/Julia.md:55]:
```suggestion
#### Step 2
```

File: lang/Julia.md [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+
+> Using VSCode
+
+- Install the Julia Extension.
+
+- Create an example file index.jl and start using Julia.
```
Comment (Panquesito7) [inline lang/Julia.md:61]:
```suggestion
- Create an example file `index.jl` and start using Julia.
```

File: lang/Julia.md [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+
+- Write your code and type `julia index.jl` in the terminal to get the output.
+
+#### On Linux
```
Comment (Panquesito7) [inline lang/Julia.md:65]:
```suggestion
### 🐧 On Linux
```

File: lang/Julia.md [hunk 12]
```diff
@@ -0,0 +1,123 @@
...
+
+- Open a new terminal and go to your Downloads folder:
+
+```
+cd ~/Downloads
+```
+
+- Use `wget` to retrieve the latest compressed [Julia Linux Binaries](https://julialang.org/downloads/):
+
+```
```
Comment (Panquesito7) [inline lang/Julia.md:75]:
```suggestion
```bash
```

File: lang/Julia.md [hunk 13]
```diff
@@ -0,0 +1,123 @@
...
+wget https://julialang-s3.julialang.org/bin/linux/x64/1.7/julia-1.7.0-linux-x86_64.tar.gz
+```
+
+- Extract the `.tar.gz`:
+
+```
```
Comment (Panquesito7) [inline lang/Julia.md:81]:
```suggestion
```bash
```

File: lang/Julia.md [hunk 14]
```diff
@@ -0,0 +1,123 @@
...
+tar -xvzf julia-1.7.0-linux-x86_64.tar.gz
+```
+
+- Copy the extracted folder to [`/opt`](https://askubuntu.com/a/34922/292615):
+
+```
```
Comment (Panquesito7) [inline lang/Julia.md:87]:
```suggestion
```bash
```

File: lang/Julia.md [hunk 15]
```diff
@@ -0,0 +1,123 @@
...
+sudo cp -r julia-1.7.0 /opt/
+```
+
+- Finally, create a symbolic link to `julia` inside the `/usr/local/bin` folder:
+
+```
```
Comment (Panquesito7) [inline lang/Julia.md:93]:
```suggestion
```bash
```

File: lang/Julia.md [hunk 16]
```diff
@@ -0,0 +1,123 @@
...
+
+- Install the Julia Extension on VSCode.
+
+- Create an example file index.jl and start using Julia.
+
+- Write your code and type `julia index.jl` in the terminal to get the output.
+
+---
+
+### Popular Julia IDEs/Editors
```
Comment (Panquesito7) [inline lang/Julia.md:105]:
```suggestion
## ✏️ Popular Julia IDEs/Editors
```

File: lang/Julia.md [hunk 17]
```diff
@@ -0,0 +1,123 @@
...
+
+- [Juno](https://junolab.org/) is the most popular IDE for the Julia language. Development focus of Juno has shifted to the [Julia extension for VSCode](https://www.julia-vscode.org/).
+- [Visual Studio Code](https://code.visualstudio.com/download)
+
+----
+
+### Some cool projects in Julia
```
Comment (Panquesito7) [inline lang/Julia.md:112]:
```suggestion
## 😎 Some cool projects in Julia
```

File: lang/Julia.md [hunk 18]
```diff
@@ -0,0 +1,123 @@
...
+
+- [CliMA](https://clima.caltech.edu/) - Climate Modeling Alliance.
+- Julia is also excelling in the pharmaceutical modeling space.
+- Lots of drug discovery is also happening using Julia fundamentally.
+- [Pumas-AI](https://pumas.ai/) developed a revolutionary platform for model based drug development which utilizes the powerful Julia programming language.
+
+---
+
+### Conclusion
```
Comment (Panquesito7) [inline lang/Julia.md:121]:
```suggestion
## 🚀 Conclusion
```

File: lang/Julia.md [hunk 19]
```diff
@@ -0,0 +1,123 @@
...
+
+If you are someone who is very interested in the climate space or the medical space then you should go for Julia.
```
Comment (Panquesito7) [inline lang/Julia.md:123]:
```suggestion
If you are someone who is very interested in the climate space or the medical space then you should go for Julia.

**Thanks for reading** ✨
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Great work! 🚀

**Comment by Shivansh-yadav13** [comment]
> Great work! 🚀

Thank you!

File: lang/Julia.md [hunk 20]
```diff
@@ -0,0 +1,117 @@
...
+author: Shivansh Yadav
+---
+```julia
+println("Hello World!")
+```
+---
+
+## 🤔 What is Julia
+
+Julia is a new, general purpose, high performance programming language with the tagline **"Looks like Python, feels like Lisp, runs like Fortran”** It is easy to   use, powerful and extremally performant.
```
Comment (Panquesito7) [inline lang/Julia.md:14]:
```suggestion
Julia is a new, general-purpose, high-performance programming language with the tagline **"Looks like Python, feels like Lisp, runs like Fortran”**. It is easy to use, powerful, and extremely performant.
```

File: lang/Julia.md [hunk 21]
```diff
@@ -0,0 +1,117 @@
...
+## 🤔 What is Julia
+
+Julia is a new, general-purpose, high-performance programming language with the tagline **"Looks like Python, feels like Lisp, runs like Fortran”**. It is easy to use, powerful, and extremely performant.
+
+---
+## 📈 Why to use Julia
+
+- When using Python and other languages in production and environments it becomes a sort of a burden and a limitation. While Julia is computationally efficient and easy to use for developers.
+
+- Syntactically Julia is quite similar to python so if you write code that does very similar stuff to what it does in python, it does faster in Julia.
```
Comment (Panquesito7) [inline lang/Julia.md:21]:
```suggestion
- Syntactically Julia is quite similar to Python, so if you write code that does very similar stuff to what it does in Python, it does faster in Julia.
```

**Review by Panquesito7 — APPROVED** [review]
For me, this is looking good 🚀 Thank you! 😄👍
CC: @amino19. Review when you have time. 🙂

**Review by amino19 — APPROVED** [review]
@Shivansh-yadav13: Thanks for your valuable contributions.
Apologize for the delay, we're involved on [bugtrons](https://github.com/bugtrons). :sparkles: 

@Panquesito7: Thank you so much for reviewing PR, & active participation in such practice, I  appreciate it!
